filter_plugins/ec2_custom.py

```diff
@@ -1,5 +1,15 @@
 from ansible import errors
 
+# list all known app component names.
+VALID_APP_COMPONENT_NAMES = ["frontend",
```

Comment:
this should be a variable that can be supplied to the filter from ansible, otherwise we have to maintain this var in the filter